README.md

```diff
@@ -34,6 +34,113 @@ Install prerequisites first: `sudo apt-get install build-essential libssl-dev li
 
     sudo pip install -U ansible
 
+#### GnuPG
+
+Make sure GNU privacy guard is installed.
+
+    sudo apt-get install gnupg
+
+Also make sure it is callable using the command 'gpg2' (not just 'gpg'). You may need a workaround
+here, e.g. creating a symlink from gpg2 to gpg.
+
+
+## Decide on a gpg keypair to use for gitian
+
+You'll be asked to (optionally) refer to a gpg key in gitian.yml.
+
+You can generate a keypair specifically for zcash gitian builds with a command like the one below.
+
+```
+gpg2 --quick-generate-key --batch --passphrase '' "Harry Potter (zcash gitian) <hpotter@hogwarts.wiz>"
```

Comment:
In documentation, `quick-generate-key` and `quick-gen-key` are described as equivalent. But `quick-gen-key` is apparently more likely to work so switched it to that.
https://www.gnupg.org/documentation/manuals/gnupg/OpenPGP-Key-Management.html

The note about the symlink is there because I had to do that on a mac after installing via homebrew. I think if you build and install GnuPG 2.x from source with the default settings, you'll get a command called `gpg`, and package managers like apt are doing something extra to name it `gpg2` in order to keep version 1.x around.